scripts/getParentsTags.py

```diff
@@ -60,41 +108,23 @@ def getCMSSWRelease( ):
     
     (options, arguments) = parser.parse_args()
     
-    #### Needs cmsrel inside a CMSSW > 80X
     theRelease = getCMSSWRelease()
-    print "- Getting CondDBFW from release",theRelease
-    connectionString=""
-
-    if isCMSSWBefore81X( theRelease ):
-        connectionString="frontier://pro/CMS_CONDITIONS"
-    else:
-        connectionString="frontier://FrontierProd/CMS_CONDITIONS"
-        
-    from CondCore.Utilities.CondDBFW import querying
-    connection = querying.connect(connectionString)
+    print "- Getting conddblib from release",theRelease
+    connectionString="frontier://FrontierProd/CMS_CONDITIONS"
 
-    payload = connection.payload(hash=options.hash)
-    my_dict = payload.parent_tags().as_dicts()
+    tags = get_parent_tags(connectionString,options.hash)
 
-    fldmap = (
-        'name',  's',
-        'synch', 's',
-        'insertion', '',
-        )
+    #pp = pprint.PrettyPrinter(indent=4)
+    #pp.pprint(tags)
 
-    # Leave these alone for unquoted, tab-delimited record format.
-    head = '\t'.join(fldmap[0:len(fldmap):2]) + '\n'
+    #print tags
+    #for tag in tags:
+    #    print tag
```

Comment:
As this is not CMSSW, I don't mind to have these commented lines in the code, but in view of integrating it in CMSSW, we should get rid of it.